index.html

```diff
@@ -17,25 +17,30 @@
     <header class="nav-container">
         <img src="./images/mobile_menu.png" class="nav-icon" alt="Nav-bar Hamburger">
         <div class="log">
-            <ul>
-                <li><a href="https://facebook.com"> <img src="images/facebook.png" alt=""></a></li>
-                <li><a href="https://twitter.com"><img src="images/twitter.png" alt=""></a></li>
-                <li><a href="#">Logout</a></li>
-                <li><a href="#">Mypage</a></li>
-                <li><a href="#">English</a></li>
-            </ul>
-        </div>
-        <nav class="nav-bar">
-            <a href="index.html"><img src="images/wmflogo.jpg" alt="logo"></a>
-            <ul>
-                <li><a href="about.html">About</a></li>
-                <li><a href="#">Program</a></li>
-                <li><a href="#">Join</a></li>
-                <li><a href="#">Sponsor</a></li>
-                <li><a href="#">News</a></li>
-            </ul>
-            <span class="campaign"><a href="#">Campaign</a></span>
-        </nav>
+            <span></span>
+            <div class="log-link-container">     
+                <ul>
+                    <li><a href="https://facebook.com"> <img src="images/facebook.png" alt=""></a></li>
+                    <li><a href="https://twitter.com"><img src="images/twitter.png" alt=""></a></li>
```

Comment:
- [ Required ] As requested by the previous reviewer, I think it would be better if these two images have descriptive values for the `alt` attribute. It's considered good practice for images to have the `alt` attribute which makes it accessible to web users accessing the web using screen readers.